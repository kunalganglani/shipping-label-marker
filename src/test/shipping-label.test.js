import React from 'react';
import renderer from 'react-test-renderer';
import ShippingLabel from '../features/shipping-label-maker/shipping-label';
test('ShippingLabel renders correctly', () => {
  const props = {
    data: {
      from: {
        name: '',
        street: '',
        city: '',
        state: '',
        zip: '',
      },
      to: {
        name: '',
        street: '',
        city: '',
        state: '',
        zip: '',
      },
      weight: {
        weightValue: '',
        weightUnit: 'Pounds',
      },
      shippingOption: '1',
    },
  };
  const tree = renderer
    .create(<ShippingLabel data={props.data} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
