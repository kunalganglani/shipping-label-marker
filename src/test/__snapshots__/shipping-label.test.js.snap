// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShippingLabel renders correctly 1`] = `
Array [
  <div
    className="css-wd9lp9"
  >
    <h2>
      Shipping Information
    </h2>
    <h3>
      Shipping Option: GROUND
    </h3>
    <h3>
      Weight:  Pounds
    </h3>
    <h3>
      Cost: $ 0.00
    </h3>
  </div>,
  <div
    className="css-wd9lp9"
  >
    <h2>
      Sender's Information
    </h2>
    <h4>
      Name: 
    </h4>
    <h4>
      Street: 
    </h4>
    <h4>
      City: 
    </h4>
    <h4>
      State: 
    </h4>
    <h4>
      Zip: 
    </h4>
  </div>,
  <div
    className="css-wd9lp9"
  >
    <h2>
      Receiver's Information
    </h2>
    <h4>
      Name: 
    </h4>
    <h4>
      Street: 
    </h4>
    <h4>
      City: 
    </h4>
    <h4>
      State: 
    </h4>
    <h4>
      Zip: 
    </h4>
  </div>,
]
`;
